nipap/debian/control

```diff
@@ -17,7 +17,7 @@ Description: Neat IP Address Planner
 
 Package: nipapd
 Architecture: all
-Depends: debconf, nipap-common, python (>= 2.7), ${misc:Depends}, python-psycopg2, python-flask, python-flask-xml-rpc, python-flask-compress, python-tornado, python-parsedatetime, python-tz, python-dateutil, python-psutil, python-pyparsing
+Depends: debconf, nipap-common, python (>= 2.7), ${misc:Depends}, python-psycopg2, python-flask, python-flask-xml-rpc, python-flask-restful, python-flask-compress, python-requests, python-tornado, python-parsedatetime, python-tz, python-dateutil, python-psutil, python-pyparsing
```

Comment:
python-requests is only needed by the test suite, right? In that case, don't list it as a dependency for nipapd.

Reply:
Right!
Change has been made in the newest commit.